docs/FAQ.md

```diff
@@ -515,6 +515,28 @@ by `jj rebase` to rebase the changes in a commit. It's used in `jj log` to
 indicate which commits are empty. It's used in the `files()` revset function
 (and by `jj log <path>`) to find commits that modify a certain path. And so on.
 
+## How do I revert a merge commit? `jj revert <merge>` does nothing
+
+Instead use `jj restore --from <first parent>` and add the respective message with
+`jj desc` or `jj commit`, since removing the parent edge to the merged commits
+removes the added trees.
```

Comment:
Maybe better to say "To revert changes merged from the second parent," or something?

> removing the parent edge to the merged commits removes the added trees

This part was difficult for me to understand, maybe because `revert` doesn't actually remove the parent edges.